.travis.yml

```diff
@@ -50,7 +50,7 @@ before_install:
   - export PATH=$HOME/.yarn/bin:$PATH
 
 install:
-  - yarn install --no-lockfile --non-interactive
+  - yarn install --no-lockfile --non-interactive --ignore-engines
```

Comment:
ignoring engines cause `@shopify/react-utilities` will fail our tests

Reply:
Tried running `yarn` on this branch on local & got,
`error @shopify/react-utilities@2.0.3: The engine "node" is incompatible with this module. Expected version ">=8.9.0".`

Setting the node version of the container to `8` should solve this. Node 8 is LTS anyway.

Reply:
yeah, I tried to keep this, but I guess there's no reason we shouldn't use 8

PS there's 6 LTS too 😁

Reply:
ah i forget they support multiple LTS' https://github.com/nodejs/Release#release-schedule